cmd/aries-js-worker/src/agent-rest-client.js

```diff
@@ -128,6 +128,12 @@ const pkgs = {
             pathParam:"id"
         },
     },
+    kms: {
+        CreateKeySet: {
+            path: "/kms/createkeyset",
```

Comment:
shouldn't need `create` prefix in a REST path.

POST to keysets should imply creation.

Reply:
I will change it

Reply:
or if there are multiple steps in creation (like in DID Exchange), then we would see something like:

https://github.com/hyperledger/aries-framework-go/blob/c18b4aed0de3022064f90d7bf2b89f40963b244b/pkg/controller/rest/didexchange/operation.go#L28

Reply:
(note the `-`separator in the existing APIs).

Reply:
I would prefer HTTP GET with /generatekeyset path as we are not POSTing anything here.

Reply:
done